app/src/main/kotlin/com/mgorski/discountasciiwarehouse/view/recyclerview/OffsetItemDecoration.kt

```diff
@@ -0,0 +1,18 @@
+package com.mgorski.discountasciiwarehouse.view.recyclerview
+
+import android.graphics.Rect
+import android.support.annotation.DimenRes
+import android.support.v7.widget.RecyclerView
+import android.view.View
+
+class OffsetItemDecoration(@DimenRes private val offsetId: Int) : RecyclerView.ItemDecoration() {
+
+    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
+        val dimensionPixelOffset = view.context.resources.getDimensionPixelOffset(offsetId)
+
+        outRect.left = dimensionPixelOffset
+        outRect.right = dimensionPixelOffset

```

Comment:
This decoration does not take into account the grid layout you're using. It ends up setting double the space between items.